simply-rets-api-helper.php

```diff
@@ -1162,6 +1162,31 @@ public static function srResidentialResultsGenerator( $response, $settings ) {
                 !empty($vendor) ? array("sr_vendor" => $vendor) : array()
             );
 
+
```

Comment:
- [ ] Move this to it's own function in `SrUtil` class.
- [ ] Conditionally show the `<br/`> that's in the markup right now.